.travis.yml

```diff
@@ -4,7 +4,8 @@ os:
   - linux
   - osx
```

Comment:
Also, why not stop testing against macOS to increase CI throughput? Setfield.jl is a pure Julia package so shouldn't it behave identically across all OSes (unless Julia has some very specific bugs)?